deploy/kubernetes/elastic-agent-managed-kubernetes.yaml

```diff
@@ -19,6 +19,7 @@ spec:
           effect: NoSchedule
       serviceAccountName: elastic-agent
       hostNetwork: true
+      hostPID: true
```

Comment:
@daniel-almeida Can we add the `why` we set that setting to true in a comment? This will help future dev and our user to understand why it's needed.

Reply:
Of course. Just did that.